pkg/oc/cli/cmd/create/deploymentconfig.go

```diff
@@ -74,10 +74,9 @@ func (o *CreateDeploymentConfigOptions) Complete(cmd *cobra.Command, f *clientcm
 		(argsLenAtDash == 0),
 		(argsLenAtDash > 1):
 		return fmt.Errorf("NAME is required: %v", args)
-
 	}
 
-	labels := map[string]string{"deployment-config.name": args[0]}
+	labels := map[string]string{"app": args[0]}
```

Comment:
To be honest we should not create any labels for the users without being told explicitly. Changing this is definite no-go because we might break some existing users. `app` per se is opinionated, and although it does make sense in the case of `oc new-app` it does not in the context of `oc create dc`.

Reply:
> Changing this is definite no-go because we might break some existing users.

This PR will change the label when users create a __new__ deploymentConfig. I didn't think it has high risks to break existing users. And, if it is difficult to remove `deployment-config.name`, it would be great that putting second `app` key value without removing `deployment-config.name`. Current cleaning up command looks not cool.

We need to clean up dc, rc and pods by this command...

```
  # oc delete all -l deployment-config.name=myapp
```

> app per se is opinionated, and although it does make sense in the case of oc new-app it does not in the context of oc create dc.

Not only `new-app`, but also `oc create deployment` and `kubectl create deployment` have the `app` key for label.

Reply:
@soltysh OK, in case removing `deployment-config.name` destroys existing users, I added remained it for the key.